lib/shopify_app/app_proxy_verification.rb

```diff
@@ -3,7 +3,7 @@ module AppProxyVerification
     extend ActiveSupport::Concern
 
     included do
-      skip_before_action :verify_authenticity_token
+      skip_before_action :verify_authenticity_token, raise: false
```

Comment:
to clarify Rails now raises if you try and skip and action that doesn't exist? In that case 👍